### PR TITLE
Show the full hierarchy on the Schools view 

### PR DESCRIPTION
**Pull request by gitumarkk**
Not just one level of Foreign keys

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: hierarchy/models.py [hunk 1]
```diff
@@ -47,3 +47,12 @@ def __unicode__(self):
 
     class Meta:
         verbose_name = "4. School"
+
+    def display_district(self):
+        return self.zone.district
+
+    def display_province(self):
+        return self.zone.district.province
```
Comment (smn) [inline hierarchy/models.py:55]: can `self.zone.district` ever be `None`?

File: hierarchy/models.py [hunk 2]
```diff
@@ -47,3 +47,12 @@ def __unicode__(self):
 
     class Meta:
         verbose_name = "4. School"
+
+    def display_district(self):
+        return self.zone.district
```
Comment (smn) [inline hierarchy/models.py:52]: can `self.zone` ever be `None`?

File: hierarchy/models.py (hunk 1 above)
Comment (imsickofmaps) [inline hierarchy/models.py:55]: No

File: hierarchy/models.py (hunk 2 above)
Comment (imsickofmaps) [inline hierarchy/models.py:52]: No

**Comment by smn** [comment]
:+1:

**Pull request merged**